src/styles/header.scss

```diff
@@ -0,0 +1,62 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-left: 50px;
+  padding-right: 50px;
+  font-family: Roboto, Arial, sans-serif;
+  text-transform: uppercase;
+  font-size: 12px;
+  font-weight: 500;
+
+  --font-color: #00acdc;
+}
+
+.header__logo-link {
+  display: flex;
+  text-decoration: none;
+}
+
+.header__nav-list {
+  list-style: none;
+  display: flex;
+}
+
+.header__nav-link {
+  position: relative;
+  text-decoration: none;
+  color: #060b35;
+  display: inline-block;
+  line-height: 60px;
+}
+
+.header__nav-link:hover {
+  color: var(--font-color);
+}
+
+.header__nav-link--active::after {
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  content: '';
+  height: 4px;
+  width: 100%;
+  background-color: var(--font-color);
+  border-radius: 8px;
+}
+
+.header__nav-link--active {
+  color: var(--font-color);
+}
+
+.header__nav-item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.is-active {
+  color: var(--font-color);
+}
```

Comment:
The class `.is-active` seems redundant if it serves the same purpose as `.header__nav-link--active`. Consider removing it or ensuring it has a distinct purpose.